configs/sa.events.config.json

```diff
@@ -4,6 +4,7 @@
 		"AlertManagerURL": "http://10.19.110.5:9093/api/v1/alerts",
 		"ResetIndex": false,
 		"Debug": true,
+		"ServiceType" : "event",
```

Comment:
I think I'd prefer to see these as the plural version. `events` and `metrics`.

Reply:
Fixed